packages/features/borrow/src/borrow-fixed/hooks/use-borrow-fixed-chart.tsx

```diff
@@ -0,0 +1,63 @@
+import { useContext } from 'react';
+import { useTheme } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import { ChartToolTipDataProps } from '@notional-finance/mui';
+import { BorrowFixedContext } from '../../borrow-fixed/borrow-fixed';
+import { useFCashMarket } from '@notional-finance/notionable-hooks';
+import {
+  getDateString,
+  formatNumberAsPercent,
+} from '@notional-finance/helpers';
+import { useMaturitySelect } from '@notional-finance/trade';
+
+export const useBorrowFixedChart = () => {
+  const theme = useTheme();
+  const {
+    state: { deposit },
+  } = useContext(BorrowFixedContext);
+  const { onSelect } = useMaturitySelect('Debt', BorrowFixedContext);
```

Comment:
You call the useMaturitySelect hook twice in the same component, once here and in the parent. Can you just return all the values from this hook instead?